packages/react-components/react-skeleton/SPEC.md

```diff
@@ -0,0 +1,101 @@
+# @fluentui/react-skeleton Spec
+
+## Background
+
+The `Skeleton` component is a temporary animation placeholder for when a service call takes time to return data and we don't want to block rendering the rest of the UI.
+
+## Prior Art
+
+### Open UI
+
+| Library    | Component Name | Spec Link                                                                       | Notes                                                                                                                                                                  |
+| ---------- | -------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Ant Design | Skeleton       | [Skeleton](https://ant.design/components/skeleton/)                             | Has an `avatar`, `button`, `list`, and `togglebutton`. Has animated and non-animated versions, and shape variants for the component skeletons(rounded or rectangular). |
+| Fast       | Skeleton       | [Skeleton](https://explore.fast.design/components/fast-skeleton)                | Has three components: `Avatar`, `List`, and `Button`. No shape variants.                                                                                               |
+| UI Fabric  | Shimmer        | [Shimmer](https://developer.microsoft.com/en-us/fluentui#/controls/web/shimmer) | Shimmer has a way to pass in custom elements as well as the defaults. There is also a prop to change the colors of the Skeleton components.                            |
+
+### Comparison of v8 and v0
+
+The existing components are:
+
+- v8
+  - `Shimmer`
+- v0
+
+## Sample Code
+
+Basic example:
+
+```jsx
+import { Skeleton, SkeletonItem } from '@fluentui/react-skeleton';
+
+function App() {
+  return (
+    <Skeleton>
+      <SkeletonItem />
+    </Skeleton>
+  );
+}
+```
+
+## Variants
+
+- There are three basic `Skeleton` shapes
+  - `Rectangle`
+  - `Circle`
+  - `Square`
+
+### Shape
+
+The `Skeleton` is a combination of any of the two shapes in order to represent the content that will be loaded onto the screen. That is, it is a combination of rectangles and circles that is a visual representation of the wireframe of a page.
+
+## API
+
+### Slots
+
+- `root` - The root slot of the `Skeleton` is the container that will contain the slots that make up a `Skeleton` and any data that the `Skeleton` will load. The default html element is a `svg`.
+
+### Props
+
+See API at [Skeleton.types.tsx]().
```

Comment:
It would also be nice to have a reference to how the sub components will work or how they will be used with Skeleton.